source/includes/_server_to_server.md

```diff
@@ -108,6 +116,18 @@ curl https://example.com/rpc/create_operation                           \
     <td colspan=2>HTTP URL</td>
     <td colspan=2>Always</td>
   </tr>
+  <tr>
+    <td colspan=2>customer_id</td>
+    <td colspan=2>User identifier</td>
+    <td colspan=2>String</td>
+    <td colspan=2>Optional</td>
```

Comment:
Is this the same as `merchant_customer_id`? If so, shouldn't it be required?

Reply:
If you insist.

Reply:
I'm not insisting. I made a question. Could you answer it, please?

Reply:
I already committed my answer.

Reply:
Ok, then **if this is meant to reflect the `merchant_customer_id` from the other flows**, I would rename it the same (`merchant_customer_id`) and you are missing to add it on the request JSON body example as now it is required.

Reply:
The example bit is fair, but changing the name to include `merchant` is imo redundant, since the other side is a merchant and there's only one customer per transaction.

Reply:
ok, as you prefer